src/language-html/printer-html.js

```diff
@@ -229,7 +230,8 @@ function genericPrint(path, options, print) {
                         })
                       : isScriptLikeTag(node) &&
                         node.parent.type === "root" &&
-                        options.parser === "vue"
+                        options.parser === "vue" &&
+                        !isVueIndentScriptAndStyleConfigured(node, options)
```

Comment:
Wouldn’t this work?

```suggestion
                        !options.vueIndentScriptAndStyle
```

Reply:
yes, it works. 
There are too much branches, making me a little bit confused, so i wrote as a function.
I will apply your changes later.